### DISCUSSION TITLE
[Feature] suggest adding a feature that groups photos by date in the album

### DISCUSSION THREAD

**Discussion by ExceptionsOccur** [immich-app/immich#14661] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

with a large number of photos in the album, it can be difficult to locate specific images easily,  so I would appreciate it if there could be a feature to group photos by date, similar to how the 'Photos' section on the homepage is displayed, thanks!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4739
